src/osx/core/uilocale.mm

```diff
@@ -185,7 +187,7 @@ int CompareStrings(const wxString& lhs, const wxString& rhs,
                     str = [m_nsloc localizedStringForCountryCode:[m_nsloc countryCode]];
                     break;
                 case wxLOCALE_FORM_ENGLISH:
-                    str = [m_nsloc displayNameForKey:NSLocaleIdentifier value:[m_nsloc localeIdentifier]];
+                    str = [enLocale displayNameForKey:NSLocaleIdentifier value:[m_nsloc localeIdentifier]];
```

Comment:
And this
```suggestion
                    str = [enLocale displayNameForKey:NSLocaleCountryCode value:[m_nsloc localeIdentifier]];
```

Reply:
Same as for language.

Although it works when using `localeIdentifier` for the value attribute it would be clearer to use `countryCode`:
```c++
str = [enLocale displayNameForKey:NSLocaleIdentifier value:[m_nsloc countryCode]];
```
Even more concise would be:
```c++
str = [enLocale localizedStringForCountryCode:[m_nsloc countryCode]];
```
Therefore I would recommend to use the latter.